gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -13,9 +13,12 @@ from google.api_core import gapic_v1                   # type: ignore
 from google.api_core import retry as retries           # type: ignore
```

Comment:
All my comments from above in the ads-template version also apply here.

Reply:
done